sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md

```diff
@@ -4,19 +4,33 @@
 
 ### Features Added
 
+- Added support for `ManagedInstanceServerConfigurationOption`, `ManagedInstanceStartStopSchedule` and `ManagedLedgerDigestUpload` resources.
+- Added `filter` and `expand` parameters to `Get` and `Exists` methods of `RestorableDroppedDatabaseCollection` and `SqlDatabaseCollection`.
+- Added `filter` and `expand` parameters to `Get` methods of `RestorableDroppedDatabaseResource` and `SqlDatabaseResource`.
+- Added `RevalidateDatabaseEncryptionProtector` and `RevertDatabaseEncryptionProtector` methods to `SqlDatabaseResource`.
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+- Upgraded API version of Instance Failover Groups to `2022-05-01-preview`.
```

Comment:
Are we planed to release a beta sdk or stable?

Reply:
Stable version. The tag is still `package-composite-v5` with some preview API versions, which is allowed exception for Sql.

Reply:
fixed